services/checks/checks.go

```diff
@@ -166,6 +157,25 @@ func (s *Service) Run(ctx context.Context) {
 	}
 }
 
+// StartChecks triggers STT checks downloading and execution. It returns services.ErrSTTDisabled if STT is disabled.
+func (s *Service) StartChecks(ctx context.Context) error {
+	settings, err := models.GetSettings(s.db)
+	if err != nil {
+		return err
+	}
+
+	if !settings.SaaS.STTEnabled {
+		return services.ErrSTTDisabled
+	}
+
+	nCtx, cancel := context.WithTimeout(ctx, checksTimeout)
```

Comment:
According to the ticket topic "API for manual checks execution" - an hour timeout looks too long to wait on error.
Shouldn't we change it here or make configurable?

Reply:
One hour is the default timeout for checks execution, however, you can reduce that time by passing `ctx` with another timeout setting(or with cancel/deadline). So the user of that function has an option how to change awaiting time.